new_builder_evaluation/app/domain/evaluation.py

```diff
@@ -158,6 +156,40 @@ def downloads_scoring_datasets(
                 final_datasets.append(final_dataset)
         return final_datasets
 
+    def get_no_scoring_datasets(self, task_id: int):
+        jsonl_no_scoring_datasets = self.dataset_repository.get_no_scoring_datasets(
+            task_id
+        )
+        return jsonl_no_scoring_datasets
+
+    def downloads_no_scoring_datasets(
+        self,
+        jsonl_all_datasets: list,
+        bucket_name: str,
+        task_code: str,
+        model: str,
+    ):
+        folder_name = model.split("/")[-1].split(".")[0]
+        final_datasets = []
+        for dataset in jsonl_all_datasets:
+            final_dataset = {}
+            base_dataset_name = "datasets/{}/{}.jsonl".format(
+                task_code, dataset["dataset"]
+            )
+            self.s3.download_file(
+                bucket_name,
+                base_dataset_name,
+                "./app/models/{}/datasets/{}.jsonl".format(
+                    folder_name, dataset["dataset"]
+                ),
+            )
+            final_dataset["dataset_type"] = "base"
```

Comment:
There are two different types of not scoring datasets: standard and hidden. I'm not sure if they are used differently at the moment, but given the differentiation, we should keep it.